pkg/cnab/provider/claims.go

```diff
@@ -1,17 +1,23 @@
 package cnabprovider
 
 import (
+	"path/filepath"
+
 	"github.com/deislabs/cnab-go/claim"
 	"github.com/deislabs/cnab-go/utils/crud"
-	"github.com/deislabs/duffle/pkg/duffle/home"
 	"github.com/pkg/errors"
 )
 
+const (
+	// ClaimsDirectory represents the name of the directory where claims are stored
+	ClaimsDirectory = "claims"
+)
+
 func (d *Duffle) NewClaimStore() claim.Store {
 	// TODO: I'm going to submit a PR after this so that GetHomeDir doesn't return an error
 	homepath, _ := d.GetHomeDir()
-	h := home.Home(homepath)
-	return claim.NewClaimStore(crud.NewFileSystemStore(h.Claims(), "json"))
+	claimsPath := filepath.Join(homepath, ClaimsDirectory)
```

Comment:
Is this something that should be moved into a function on the Config struct like 

https://github.com/deislabs/porter/blob/94a5452744191574909e5b5249c39b4fd2958f7b/pkg/config/config.go#L121

Reply:
Good call; added in most recent commit